internal/pkg/template/diff/diff.go

```diff
@@ -11,6 +11,11 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// For a property whose value is a list of map, map its name to the identifying property of the items.
+var mapItemsIdentifiedBy = map[string]string{
+	"ContainerDefinitions": "Name",
```

Comment:
niceeeeee

Reply:
though i wonder (nit) if something like `seqObjectIdentifiers` indicates better that it's only used to identify sequence objects? i don't feel too strongly abt it